### ISSUE TITLE
train_from_largetext_file doesn't work with prefix

### ISSUE THREAD

**Issue by WhereAmO** [minimaxir/textgenrnn#237]
Whenever I run a trained model, specifically one trained using the "largetext" augment, and generate with it using a prefix added in, it completely ignores the prefix and just goes on its own. I've seen others reporting similar issues. Also, is there any way to train on a folder of .txt files? As in multiple? That'd be a good enough workaround but I can't find any way to do that in the slightest. It seemingly only wants to accept a single .txt file. If I just hand it the location of the folder I get a permission denied error. Any help?

**Comment by noob10293** [comment]
I don't think it classifies folder locations as txt files. most text generators use the prefix, but outside of the generated text. it only outputs the text it generated after the prefix. basically, the generated text doesn't include the prefix, but it's a prediction of what comes next after the prefix
